api4/license.go

```diff
@@ -210,3 +213,35 @@ func requestTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func requestRenewalLink(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Should we also check if they are licensed before continuing the license renewal link logic?

Reply:
The `GenerateRenewalToken` checks for license